ethers-contract/ethers-contract-abigen/src/util.rs

```diff
@@ -84,6 +84,7 @@ where
 }
 
 /// Perform an HTTP GET request and return the contents of the response.
+#[allow(unused)]
```

Comment:
i think this is probably the right fix, instead of silencing the warning?

```suggestion
#[cfg(not(target_arch = "wasm32"))]
```